src/solver.cpp

```diff
@@ -585,6 +585,11 @@ inline SolverRegistrar::SolverRegistrar(IdRegistryData& registry)
              Primitive::Fusion,
              fusion::ConvCKIgemmFwdBiasActivFused{}.SolverDbId(),
              miopenConvolutionAlgoImplicitGEMM);
+    Register(registry,
```

Comment:
New solvers must be added to the end of the list.

Reply:
Done.